Marlin/src/lcd/ultralcd.cpp

```diff
@@ -3668,15 +3797,15 @@ void lcd_quick_feedback(const bool clear_buttons) {
     // M203 / M205 Velocity options
     void lcd_advanced_velocity_menu() {
       START_MENU();
-      MENU_BACK(MSG_ADVANCED_SETTINGS);
+      MENU_BACK(MSG_BACK);
 
       // M203 Max Feedrate
       MENU_MULTIPLIER_ITEM_EDIT(float3, MSG_VMAX MSG_A, &planner.max_feedrate_mm_s[A_AXIS], 1, 999);
       MENU_MULTIPLIER_ITEM_EDIT(float3, MSG_VMAX MSG_B, &planner.max_feedrate_mm_s[B_AXIS], 1, 999);
       MENU_MULTIPLIER_ITEM_EDIT(float3, MSG_VMAX MSG_C, &planner.max_feedrate_mm_s[C_AXIS], 1, 999);
 
       #if ENABLED(DISTINCT_E_FACTORS)
-        MENU_MULTIPLIER_ITEM_EDIT(float3, MSG_VMAX MSG_E, &planner.max_feedrate_mm_s[E_AXIS + active_extruder], 1, 999);
+        MENU_MULTIPLIER_ITEM_EDIT(float3, MSG_VMAX MSG_E " " MSG_ACTIVE, &planner.max_feedrate_mm_s[E_AXIS + active_extruder], 1, 999);
```

Comment:
This is not compatible with translations, because different languages use different word orders.